src/pages/Blog.m.css

```diff
@@ -1,3 +1,3 @@
 .root {
-	padding: var(--content-top-padding);
+	width: 1000px!important;
```

Comment:
Is there a specific reason we're using 1000px and does it have to be `!important`?

Reply:
1000? no, it just looks good. Important? Yes, as with JS off the CSS loading order gets messed up resulting in the Landing widget's default with taking back over.